src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py

```diff
@@ -14,3 +14,4 @@
 
 CLASSIC_REGISTRY_SKU = ['Classic']
 MANAGED_REGISTRY_SKU = ['Basic', 'Standard', 'Premium']
+SUPPORTED_OS_TYPES = ['linux', 'windows']
```

Comment:
Not needed once you reuse the existing `os_type` definition.

Reply:
Ideally, this should be reflected from an enum in the SDK. If not, if you have an existing definition, please reuse that as @djyou suggests.

Reply:
Yes the existing definition takes the values from SDK enums.